backend/app.py

```diff
@@ -1,23 +1,41 @@
 from flask import Flask, jsonify, request
 import psycopg2
 
+
 app = Flask(__name__)
+logger = app.logger
+# TODO: get from docker compose env
+logger.setLevel("INFO")
+
 
 # Connect to PostgreSQL
+# TODO: update in try catch block and add error handling
```

Comment:
Added some TODOs for future improvement. Recommend checking those out along with the additional comments added 👍🏾

Reply:
Thanks Byron. I'll be working on it this week!